packages/components/src/List/Toolbar/ColumnChooserButton/columnChooser.propTypes.js

```diff
@@ -0,0 +1,12 @@
+import PropTypes from 'prop-types';
+
+const columnChooserPropTypes = PropTypes.shape({
```

Comment:
Disturbing naming again (already discussed), you shape the columns but `columnChooser` means the chooser, not the columns :/